packages/solar-theme/src/custom-elements/ef-calendar.less

```
@@ -114,6 +114,10 @@
     bottom: 0;
     right: 0;
     left: 0;
+
+    ::slotted(*) {
```

If result is the same, it might easier to reading with `&`.
```suggestion
    &::slotted(*) {
```